VWOSdk/Core/VWOClient.cs

```diff
@@ -624,37 +624,99 @@ public bool FlushEvents()
+                integrationsMap.Add("groupName", groupName);
+                List<BucketedCampaign> campaignList = CampaignHelper.getGroupCampaigns(this._settings, Convert.ToInt32(groupId));
+                if (campaignList.Count == 0)
+                    return new UserAllocationInfo();
+                if (checkForStorageAndWhitelisting(apiName, campaignList, groupName, campaign, userId, null,
+                    shouldTrackReturningUser, variationTargetingVariables, customVariables, userStorageData = null, true))
+                {
+                    LogInfoMessage.CalledCampaignNotWinner(file, campaignKey, userId, groupName.ToString());
+                    return new UserAllocationInfo();
+                }
+                Dictionary<string, dynamic> processedCampaigns = getEligibleCampaigns(campaignList, userId, apiName, customVariables);
+                processedCampaigns.TryGetValue("eligibleCampaigns", out dynamic eligibleCampaigns);
+                processedCampaigns.TryGetValue("inEligibleCampaigns", out dynamic inEligibleCampaigns);
+                StringBuilder eligibleCampaignKeys = new StringBuilder();
+                foreach (BucketedCampaign eachCampaign in eligibleCampaigns)
+                {
+                    eligibleCampaignKeys.Append(eachCampaign.Key).Append(", ");
+                }
+                StringBuilder inEligibleCampaignKeys = new StringBuilder();
+                foreach (BucketedCampaign eachCampaign in inEligibleCampaigns)
+                {
+                    inEligibleCampaignKeys.Append(eachCampaign.Key).Append(", ");
+                }
+                string finalInEligibleCampaignKeys = inEligibleCampaignKeys.ToString();
+                string finalEligibleCampaignKeys = eligibleCampaignKeys.ToString();
+
+                LogDebugMessage.GotEligibleCampaigns(file, groupName.ToString(), userId, finalEligibleCampaignKeys.TrimEnd(new char[] { ',', ' ' }), finalInEligibleCampaignKeys.TrimEnd(new char[] { ',', ' ' }));
+                LogInfoMessage.GotEligibleCampaigns(file, groupName.ToString(), userId, ((List<BucketedCampaign>)eligibleCampaigns).Count.ToString(), campaignList.Count.ToString());
+                if (((List<BucketedCampaign>)eligibleCampaigns).Count == 1)
+                {
+                    return evaluateTrafficAndGetVariation(((List<BucketedCampaign>)eligibleCampaigns)[0], userStorageMap, customVariables, campaignKey, apiName,
+                        variationTargetingVariables, initIntegration, userId);
+                }
+                else if (((List<BucketedCampaign>)eligibleCampaigns).Count > 1)
+                {
+                    return normalizeAndFindWinningCampaign((List<BucketedCampaign>)eligibleCampaigns, campaign, userStorageMap,
+                        customVariables, campaignKey, apiName, variationTargetingVariables, initIntegration, userId, groupName.ToString(), groupId.ToString(), true);

```

Comment:
passing `true` here will not print the logs inside `normalizeAndFindWinningCampaign` method. I think we should pass false here